src/storage/txn/actions/check_txn_status.rs

```diff
@@ -99,6 +110,9 @@ pub fn check_txn_status_missing_lock<S: Snapshot>(
                 }
                 .into());
             }
+            if resolving_pessimistic_lock {
+                return Ok(TxnStatus::LockNotExistDoNothing);
+            }
```

Comment:
Should it be before L106? In this case, I think it should return `LockNotExistDoNothing` instead of `TxnNotFound` error.

Reply:
For resolving pessimistic lock, the `rollback_if_not_exist` will not be set at first and `txnNotFound` error is returned.
If we get a `txnNotFound` error resolving a secondary pessimistic lock, we could keep waiting until the seconrary lock itself expired(maybe we remove lock primary pessimistic lock first in the future), and then the `LockNotExistDoNothing ` will be returned, and we'll pessimistic rollback the secondary pessimistic lock.

Reply:
Got it.